src/views/Incorporation/IncorporationReviewConfirm.vue

```diff
@@ -315,6 +315,10 @@ export default class IncorporationReviewConfirm extends Vue {
 
   /** Is true when the certify conditions are not met. */
   get isCertifyInvalid (): boolean {
+    // Set the Court Order section valid, when it is not visible
+    if (!(this.isBaseCompany && this.isRoleStaff)) {
+      this.setCourtOrderValidity(true)
+    }
```

Comment:
Having a computed (getter) set a value in the store is counter-intuitive.

Try a solution here instead:
https://github.com/bcgov/business-create-ui/blob/0388fcd7ae1f1b618b184677856ae23ae432679f/src/store/store.ts#L775

Reply:
Updated

Reply:
LGTM.